includes/header_template.php

```diff
@@ -14,17 +14,17 @@
 				<?php
 				if ($USER_DISPLAY_NAME) {
 					?>
-					<span>
+					<div style="margin-bottom: 0.75rem">
```

Comment:
Let's move away from hard-coding new styling changes in file: let's move these over to main.css or some more local .css file.